File: Java/Assignment 10/TestingSystem_Assignment_10/src/com/vti/entity/Position.java
package com.vti.entity;

public class Position {
	private int id;
	private String name;

	/**
	 * @return the id
	 */
	public int getId() {
		return id;
	}

	/**
	 * @param id the id to set
	 */
	public void setId(int id) {
		this.id = id;
	}

	/**
	 * @return the name
	 */
	public String getName() {
		return name;
	}

	/**
	 * @param name the name to set
	 */
	public void setName(String name) {
		this.name = name;
	}

	public Position(int id, String name) {
		super();
		this.id = id;
		this.name = name;
	}

	@Override
	public String toString() {
		return "Position [id=" + id + ", name=" + name + "]";
	}

}

File: ReactJS/TypeScript/LAB2_TypeScript/Position.ts
class Position {
  private id: number;
  private name: string;

  constructor(id: number, name: string) {
    this.id = id;
    this.name = name;
  }

  public getId(): number {
    return this.id;
  }

  public getName(): string {
    return this.name;
  }

  public setId(value: number) {
    this.id = value;
  }

  public setName(value: string) {
    this.name = value;
  }
}
function printInfoPosition(position: Position): void {
  console.log(
    "ID: " + position.getId() + " Position Name: " + position.getName()
  );
}

export { Position, printInfoPosition };
